dts/bindings/pwm/ti,cc13xx-cc26xx-timer-pwm.yaml

```diff
@@ -69,4 +69,6 @@ properties:
     required: true
 
 pwm-cells:
-- period
+  - channel
+  - period
```

Comment:
why is this needed, since period can already be controlled by the PWM API?

Reply:
[`PWM_DT_SPEC_GET`](https://docs.zephyrproject.org/latest/hardware/peripherals/pwm.html#c.PWM_DT_SPEC_GET) and related macros (used in the example as well as a lot of external zephyr code like Arduino module for zephyr) consider channel and period to be required properties.

Also, period was already present, I added channel.

Reply:
This looks correct to me.